src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -309,15 +309,15 @@ def _execute_command(kwargs):
             else:
                 return result
         except ClientException as client_exception:
-            log_telemetry('client exception', log_type='trace')
+            telemetry.set_exception(client_exception, 'client exception')
```

Comment:
Sometimes, you have string first, exception second.
Here, you have it the other way around.
Is order important here?